metadata/metadata.go

```diff
@@ -229,23 +231,25 @@ func (a *Attributes) UnmarshalJSON(b []byte) error {
 	}
 	// Unmarshal to literal JSON types before doing anything else.
 	type inner struct {
-		BlockProjectKeys      string      `json:"block-project-ssh-keys"`
-		Diagnostics           string      `json:"diagnostics"`
-		DisableAccountManager string      `json:"disable-account-manager"`
-		DisableAddressManager string      `json:"disable-address-manager"`
-		EnableDiagnostics     string      `json:"enable-diagnostics"`
-		EnableOSLogin         string      `json:"enable-oslogin"`
-		EnableWindowsSSH      string      `json:"enable-windows-ssh"`
-		EnableWSFC            string      `json:"enable-wsfc"`
-		OldSSHKeys            string      `json:"sshKeys"`
-		SSHKeys               string      `json:"ssh-keys"`
-		TwoFactor             string      `json:"enable-oslogin-2fa"`
-		SecurityKey           string      `json:"enable-oslogin-sk"`
-		RequireCerts          string      `json:"enable-oslogin-certificates"`
-		WindowsKeys           WindowsKeys `json:"windows-keys"`
-		WSFCAddresses         string      `json:"wsfc-addrs"`
-		WSFCAgentPort         string      `json:"wsfc-agent-port"`
-		DisableTelemetry      string      `json:"disable-guest-telemetry"`
+		BlockProjectKeys        string      `json:"block-project-ssh-keys"`
+		Diagnostics             string      `json:"diagnostics"`
+		DisableAccountManager   string      `json:"disable-account-manager"`
+		DisableAddressManager   string      `json:"disable-address-manager"`
+		EnableDiagnostics       string      `json:"enable-diagnostics"`
+		EnableOSLogin           string      `json:"enable-oslogin"`
+		EnableWindowsSSH        string      `json:"enable-windows-ssh"`
+		EnableWSFC              string      `json:"enable-wsfc"`
+		OldSSHKeys              string      `json:"sshKeys"`
+		SSHKeys                 string      `json:"ssh-keys"`
+		TwoFactor               string      `json:"enable-oslogin-2fa"`
+		SecurityKey             string      `json:"enable-oslogin-sk"`
+		RequireCerts            string      `json:"enable-oslogin-certificates"`
+		WindowsKeys             WindowsKeys `json:"windows-keys"`
+		WSFCAddresses           string      `json:"wsfc-addrs"`
+		WSFCAgentPort           string      `json:"wsfc-agent-port"`
+		DisableTelemetry        string      `json:"disable-guest-telemetry"`
+		DisableHTTPSMdsSetup    string      `json:"disable-https-mds-setup"`
+		HTTPSMDSSkipNativeStore string      `json:"https-mds-skip-native-cert-store"`
```

Comment:
is this consistent with what we are prepping for mds? isn't the key to be named `https-mds-native-cert-store`?

Reply:
as discussed offline we'll update the MDS with this new key